### DISCUSSION TITLE
How to do text/title elements?

### DISCUSSION THREAD

**Discussion by zackradisic** [AcademySoftwareFoundation/OpenTimelineIO#1570] [Q&A]
How can I use this API to get text/title elements? Particularly looking to export to fcpxml

**Comment by jminor** [comment]
There's isn't an OTIO schema for text, titles, or subtitles yet. There is some prior discussion here: https://github.com/AcademySoftwareFoundation/OpenTimelineIO/issues/62

Are you aiming for closed captions or styled text overlays (like a title card, etc.)?
